scrape/histology/histology/spiders/histology_spider.py

```diff
@@ -14,8 +36,59 @@ def start_requests(self):
             yield scrapy.Request(url=url, callback=self.parse)
 
     def parse(self, response):
-        print("The response is here: %s" % response.url)
-        links = self.extractor.extract_links(response)
+        if (self.match_lab_topic.search(response.url)):
```

Comment:
@sdreher @nikolas This seems like a not clear way of handling this. Is there a better pattern to follow?

Reply:
I think this is a good approach for exporting the markdown content. imho, each of these MT sites are unique enough that creating a generic/abstracted exporter will not yield a good roi. I'll go ahead and merge.